Gcpe.Hub.API/Startup.cs

```diff
@@ -73,13 +80,13 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     Version = "Alpha",
                     Title = "BC Gov Hub API service",
-                    Description = "The .net Core 2.1 API for the Hub"
+                    Description = "The .Net Core API for the Hub"
                 });
             });
 
             services.AddHealthChecks(checks =>
             {
-                // checks.AddSqlCheck("Gcpe.Hub", Configuration["HubDbContext"]);
+                checks.AddSqlCheck("Gcpe.Hub", Configuration["HubDbContext"]);
                 // checks.AddUrlCheck("https://github.com");
                 checks.AddCheck("Webserver is running", () => HealthCheckResult.Healthy("Ok"));
```

Comment:
We can remove this health check if you wish and keep the DB one as this is how its currently done in the News webapp